.travis.yml

```diff
@@ -39,14 +43,12 @@ before_install:
   -  sh -e /etc/init.d/xvfb start
 
 script:
-  - pub run dependency_validator -i browser,coverage,dart_style,dartdoc,test
-  - pub run dart_dev format --check
+  - pub run dependency_validator -i browser,coverage,dart_style,test
+  - pub run dart_dev dart2-only -- format --check
```

Comment:
we're only running format in dart 2?

Reply:
Yes because pub get resolves to a newer version of dart_style on Dart 2 and the results are slightly different than with the version we get on Dart 1. So I chose to format it on Dart 2 and only perform the check on Dart 2.